experiments/tf_trainer/common/tfrecord_input.py

```diff
@@ -50,42 +51,40 @@ def _input_fn_from_file(self, filepath: str) -> types.FeatureAndLabelTensors:
     dataset = tf.data.TFRecordDataset(filepath)  # type: tf.data.TFRecordDataset
 
     parsed_dataset = dataset.map(
-        self._read_tf_example,
-        num_parallel_calls=multiprocessing.cpu_count())
+        self._read_tf_example, num_parallel_calls=multiprocessing.cpu_count())
 
-    padded_shapes=(
-            {self._text_feature: [None],
-            'sequence_length': []},
-            {label: [] for label in self._labels})
+    padded_shapes = ({
+        self._text_feature: [None],
+        'sequence_length': []
+    }, {label: [] for label in self._labels})
     # TODO(fprost): Set a filter to remove long sentences (for training).
+    parsed_dataset = parsed_dataset.filter(
```

Comment:
@fprost is this roughly what you were thinking? This doesn't allow different options for training vs not but I need something to reduce seq length for now.

Reply:
Yes, it is. Thank you very much for your patch, you can delete my TODO!